.github/workflows/release.yml

```diff
@@ -50,3 +50,8 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           folder: ./site/dist
+          # Preserve the preview deployment folder and custom grad-connection site
+          clean-exclude: |
+            preview/*
+            grad-connection/*
```

Comment:
Planning to host the `grad-connection` branch on a custom folder path (like what we had on surge.sh previously).

```
https://seek-oss.github.io/braid-design-system/grad-connection/
```

This will provide a persistent URL for the team.